modules/swagger-codegen/src/main/resources/Java/libraries/jersey2/ApiClient.mustache

```diff
@@ -526,8 +533,8 @@ public class ApiClient {
           serviceException = response.readEntity(ServiceException.class);
           serviceException.setResponseHeaders(responseHeaders);
         } catch (RuntimeException e) {
-          // e.printStackTrace();
-        }
+          throw ApiException.builder().code(statusCode).responseHeaders(responseHeaders).throwable(e).build();
```

Comment:
I'm not finding a way to get the response body as a string without needing another try/catch around getting the entity